Archives/ClmFile.cpp

```diff
@@ -456,10 +398,31 @@ namespace Archives
 	{
 		std::vector<std::string> strippedExtensions;
 
-		for (std::string path : paths) {
+		for (std::string& path : paths) {
 			strippedExtensions.push_back(XFile::ChangeFileExtension(path, ""));
 		}
 
 		return strippedExtensions;
 	}
+
+
+	// A null terminator (\0) is automatically assigned to the end of the string when placing it within the std::array
+	const std::array<char, 32> standardFileVersion { "OP2 Clump File Version 1.0\x01A\0\0\0\0" };
+	const std::array<char, 6> standardUnknown { 0, 0, 0, 0, 1, 0 };
+
+	ClmFile::ClmHeader::ClmHeader(WaveFormatEx waveFormat, uint32_t packedFilesCount) {
+		std::memcpy(fileVersion.data(), standardFileVersion.data(), fileVersion.size());
```

Comment:
I believe `std::array` allows copy through assignment:
```
fileVersion = standardFileVersion;
```

Reply:
I'll try this out since it would be easier to read. than using memcpy.